kafka-client/src/test/java/dev/responsive/kafka/testutils/ResponsiveExtension.java

```diff
@@ -51,12 +55,32 @@ public class ResponsiveExtension implements BeforeAllCallback, AfterAllCallback,
       .withEnv("KAFKA_GROUP_MIN_SESSION_TIMEOUT_MS", "1000")
       .withEnv("KAFKA_GROUP_MAX_SESSION_TIMEOUT_MS", "60000")
       .withReuse(true);
+  public static MongoDBContainer mongo = new MongoDBContainer(TestConstants.MONGODB);
 
   public static Admin admin;
 
+  public StorageBackend backend = StorageBackend.CASSANDRA;
+
+  public ResponsiveExtension() {
+  }
+
+  public ResponsiveExtension(final StorageBackend backend) {
+    this.backend = backend;
```

Comment:
we should consider extending this approach to allow for spinning up any combination of containers to speed up runs (e.g. run just with Kafka or just with Cassandra) for certain tests